shark_turbine/kernel/wave/wave.py

```diff
@@ -15,7 +15,7 @@
 )
 from .codegen import WaveEmitter
 from .expansion import expand_graph
-from ..lang import Grid
+from ..lang import Grid, IndexMapping
```

Comment:
is this import used anywhere here?

Reply:
It needed for `tkw.IndexMapping` to work.